src/govuk/components/accordion/accordion.test.js

```diff
@@ -118,13 +118,24 @@ describe('/components/accordion', () => {
         expect(expandedState).toEqual(expandedStateAfterRefresh)
       })
 
-      describe('"+/-" icon', () => {
-        it('should display "+/-" in the section headings', async () => {
+      describe('"↓/↑" icon', () => {
+        it('should display "↓/↑" in the section headings', async () => {
           await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
 
           const numberOfExampleSections = 2
 
-          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion__icon').length)
+          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion-nav__chevron').length)
+          expect(numberOfIcons).toEqual(numberOfExampleSections)
+        })
+      })
+
+      describe('hidden text', () => {
```

Comment:
It would be great if we could add a test for the "Show" text in the button as well.

Reply:
Theres's now a test for the text in it's entirely - hopefully enough?